server/src/models/Review.js

```diff
@@ -11,6 +11,7 @@ class Review extends Model {
       required: ["content"],
       properties: {
         content: { type: "string" },
+        rating: { type: ["string", "integer"], minimum: 1, maximum: 10 },
```

Comment:
I don't think this will give you the error messages you're expecting - try to create bad data in the yarn console that is outside of this range, both as integer values and string integer values. Check the JSON schema docs for how you can specify validations for multiple data types